tests/subsys/suit/cache_pool_digest/testcase.yaml

```diff
@@ -1,3 +1,5 @@
+common:
+  sysbuild: true
```

Comment:
In theory, you can limit the change to the tests that define `nrf54h20dk/nrf54h20/cpuapp` as platform, but I agree that making it consistent across all tests has some advantages (i.e. common build directory structure).

Reply:
There are issues to build tests projects for nrf52840. I think I will update it as you suggest and enable sysbuild only for nRF54H20 for now, and later we can get back to those issues when sysbuild will be enabled globally for all tests.